Workbooks/Virtual Machines - Network Dependencies/Connections Overview - Single VM/Connections Overview.workbook

```diff
@@ -1552,8 +1552,8 @@
           {
             "type": 1,
             "content": {
-              "json": "\r\n<h1>Onboard to Azure Monitors for VM</h1>\r\n<h3>The VM selected is currently not onboarded to Azure Monitors for VMs. Please click the below button to onboard.</h3>\r\n",
-              "style": "upsell"
+              "json": "\r<h3>No data was found in the VMConnection table for this machine. This workbook requires VM Insights to be enabled with processes and dependencies configured. Please click the below button to onboard to VM Insights or update your configuration.</h3>\r\n",
```

Comment:
for future reference, simpler to use markdown style `###` or whatever instead of having to have matching start and end `H3` tags